### DISCUSSION TITLE
Please bring back Send Command

### DISCUSSION THREAD

**Discussion by kjake** [timmo001/system-bridge#1929] [Q&A] [answered]
### Description

With the 2.x.x releases, I used system_bridge.send_command from Home Assistant to shutdown my computer.  With the removal of this functionality from the 3.x.x releases, System Bridge is no longer useful to me.

**Comment by timmo001** [comment, marked as answer]
This command is available, but as a power command instead.

This feature has a [PR](https://github.com/home-assistant/core/pull/72867) open to add to Home Assistant, but in the meantime, you could use a [rest command](https://www.home-assistant.io/integrations/rest_command/).

The documentation for this API was missing. I have added this recently:

https://system-bridge.timmo.dev/docs/api?id=power

**Reply by kjake** [reply]
Awesome!  Thank you!

**Comment by holantomas** [comment]
Hi, is Send command in roadmap? I need to send some custom commands and cannot find any discussion why it was removed.

**Reply by aniceberg** [reply]
Ditto! I'm looking to use Home Assistant to trigger system commands to the linked system whenever the target app needs to be restarted, etc

**Reply by timmo001** [reply]
The main reason was security and there was a rewrite of the app. What you can do, is create a script and use open path. Here is how to call a script in windows. The same can be done for any executable file in any system:

```yaml
service: system_bridge.open_path
data:
  bridge: 123456789
  path: H:\Documents\scripts\script.cmd
```

**Reply by aniceberg** [reply]
Thank you!

Does 'open path' run the script on the host system (RPi in my case) at the same user/group level that systembridge is running?

From the systembridge logs I checked, I found that I have to run systembridge at root level on the RPi in order for it to execute the keypress commands on the host system.

**Reply by holantomas** [reply]
Didn't know that it can run executables. I thought that reason was security and want to suggest something similar. Thanks.

**Reply by timmo001** [reply]
It does run under the same user yes, but your script could probably run as another user with something like this:  https://www.cyberciti.biz/open-source/command-line-hacks/linux-run-command-as-different-user/
